### ISSUE TITLE
Can apb push check whether the apb gets listed?

### ISSUE THREAD

**Issue by cunningt** [fusor/ansible-playbook-bundle#216]
I hit a nasty issue with the registry-route addon (https://github.com/minishift/minishift/issues/1933) where my apb image was not being pushed by docker correctly.    apb push however was reporting success.

I don't know if this is possible or not - but can apb push verify that the apb was listed in the catalog before reporting success/failure?    If apb push succeeds, and the user goes to the catalog and can't find their apb, they are going to be pretty confused.

**Comment by rthallisey** [comment]
> can apb push verify that the apb was listed in the catalog before reporting success/failure?
>
Yes it can. That's a really good idea for a verification @cunningt. There have been several issues reported in the past about an apb not appearing in the catalog after an ```apb push```. I think this is worth adding.
